spec/changelog.yml

```diff
@@ -2,11 +2,11 @@
 ## This file documents changes in the package specification. It is NOT a package specification file.
 ## Newer entries go at the bottom of each in-development version.
 ##
-- version: 2.5.2-next
+- version: 2.6.0
```

Comment:
I'd prefer to keep 2.5.2-next version in changelog for now. There could be other branches merged before creating the next release version.